apps/framework-cli/src/framework/controller.rs

```diff
@@ -56,6 +111,8 @@ pub fn framework_object_mapper(
             s.name.as_str(),
         ),
         original_file_path: original_file_path.to_path_buf(),
+        version: version.to_string(),
+        schema_hash: String::new(),
```

Comment:
I am a bit confused, if this is can be empty - should the type here of `schema_hash` be `optional`?

Since we have different ways to compute the hash should we even cache it?